library/system/test/etc_fstab_entry_test.rb

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/rspec
+#
+# Unit test for EtcFstab::Entry
+#
+# (c) 2017 Stefan Hundhammer <Stefan.Hundhammer@gmx.de>
+#     Donated to the YaST project
+#
+# Original project: https://github.com/shundhammer/ruby-commented-config-file
+#
+# License: GPL V2
+#
+
+require_relative "test_helper"
+require "yast2/etc_fstab"
+
+describe EtcFstab::Entry do
+  describe "#new" do
+    it "can be created empty" do
+      entry = described_class.new
+      expect(entry).not_to be_nil
+      expect(entry.to_a).to eq [nil, nil, nil, [], 0, 0]
+    end
+
+    it "can be created from an array" do
+      mount_opts = []
+      arr = ["/dev/sdb3", "/data3", "btrfs", mount_opts, 7, 2]
+      entry = described_class.new(arr)
+      expect(entry).not_to be_nil
+      expect(entry.to_a).to eq arr
+    end
+
+    it "can be created with varargs" do
+      entry = described_class.new("/dev/sdb3", "/data3", "btrfs")
+      expect(entry).not_to be_nil
+      expect(entry.to_a).to eq ["/dev/sdb3", "/data3", "btrfs", [], 0, 0]
+    end
+
+    it "can be created from a hash" do
+      args =
+        {
+          device:         "/dev/vda2",
+          mount_point:    "/home",
+          fs_type:        "xfs",
+          mount_opts:     ["foo", "bar"],
+          dump_pass:      42,
+          fsck_pass:      3,
+          comment_before: ["# Home", "# sweet home"]
+        }
+
+      entry = described_class.new(args)
+      expect(entry).not_to be_nil
+      expect(entry.to_a).to eq ["/dev/vda2", "/home", "xfs", ["foo", "bar"], 42, 3]
+      expect(entry.comment_before).to eq ["# Home", "# sweet home"]
+    end
+
+    it "can be created with named parameters" do
+      entry = described_class.new(device: "/dev/sda4", mount_point: "/data", fs_type: "ext4")
+      expect(entry).not_to be_nil
+      expect(entry.to_a).to eq ["/dev/sda4", "/data", "ext4", [], 0, 0]
+    end
+  end
+
+  describe "#parse" do
+    subject { EtcFstab::Entry.new }
```

Comment:
This is the default, no need for this line... Check also the other places.

Reply:
Removed.